tools/make-system-bundle.js

```diff
@@ -4,8 +4,11 @@ var addLicenseToFile = require('./add-license-to-file');
 
 var config = {
   baseURL: 'dist',
+  meta: {
+    'symbol-observable': { build: false }
```

Comment:
Since this is external dependency, I've marked to exclude in systemjs builder to not to include it and resolve it externally. I'm not daily user of systemjs, so not sure if this is correct - sending out changes to see if there's correct recommendation if I did wrong.